File: evals/fixtures/java_non_ascii/src/main/java/com/örnek/controller/SiparişController.java
package com.örnek.controller;

import com.örnek.service.ÖdemeService;

// Siparişi işler — ödeme servisini çağırır.
// Byte-vs-char slicing bug'ı olsa: class adı, method adı ve
// "new ÖdemeService()" çağrısı mangle olurdu.
public class SiparişController {
    private final ÖdemeService ödeme;

    public SiparişController() {
        this.ödeme = new ÖdemeService();
    }

    // Gerçek iş metodu: import edilen servise cross-file çağrı atar.
    public String işle(int tutar) {
        String sonuç = ödeme.ödemeAl(tutar);
        return sonuç;
    }
}

File: evals/fixtures/kotlin_non_ascii/src/SiparişController.kt
package com.örnek.controller

import com.örnek.service.ÖdemeService
import com.örnek.service.Günlükçü

// Sipariş controller — ödeme servisine cross-file çağrı.
// Byte-vs-char slicing bug'ı olsa class adı mangle olurdu.
class SiparişController {
    private val ödeme = ÖdemeService()
    private val günlük = Günlükçü()

    fun işle(tutar: Int): String {
        val sonuç = ödeme.ödemeAl(tutar)
        günlük.info("işlem tamam")
        return sonuç
    }
}
